policy/support/obj_perm_sets.spt

```diff
@@ -170,6 +170,7 @@ define(`create_file_perms',`{ getattr create open }')
 define(`rename_file_perms',`{ getattr rename }')
 define(`delete_file_perms',`{ getattr unlink }')
 define(`manage_file_perms',`{ create open getattr setattr read write append rename link unlink ioctl lock }')
+define(`manage_metadata_file_perms',`{ create getattr setattr rename link unlink }')
```

Comment:
NAK. I don't think "metadata" fits in the name, here or in any of the other permission set and pattern additions.

Reply:
Removed